packages/desktop/views/update-stronghold/views/UpdateStrongholdView.svelte

```diff
@@ -16,12 +18,16 @@
     let isBusy = false
 
     async function onSubmit(): Promise<void> {
+        const onboardingType = $onboardingProfile.onboardingType
+
         try {
             isBusy = true
             if (isRecovery) {
                 await migrateStrongholdFromOnboardingProfile(password)
+                Platform.trackEvent('stronghold-migration', { success: true, onboardingType })
```

Comment:
We should add here a conditional if `features.analytics.strongholdMigration.enabled`